packages/rs-dpp/src/data_contract/document_type/methods/mod.rs

```diff
@@ -216,7 +215,9 @@ impl DocumentTypeV0Methods for DocumentTypeV0 {
     }
 
     fn requires_revision(&self) -> bool {
-        self.documents_mutable || self.documents_transferable != Transferable::Never
+        self.documents_mutable
+            || self.documents_transferable.is_transferable()
+            || self.trade_mode.seller_sets_price()
```

Comment:
Well no, because the revision is only required when the document changes, which happens when the seller sets the price, in a bidding system, it would be tradable, but the document itself wouldn't change because there would be no price updating.